components/mintmaker/base/osv-database-builder/create-database-pvc.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
The default storage class provides EBS volumes. The pvc is going to be bounded to a single node, it means that all the jobs that will use it would need to be scheduled on the same node, this can cause a bottleneck.

Reply:
not at this moment.